CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Minikube Release Notes
 
+## Version 0.16.0 - 2/2/2017
+* Updated minikube ISO to v1.0.6
```

Comment:
I linked it to the 0.16 tag path.  This doesn't exist yet but will after I merge this PR and tag it